Cargo.toml

```diff
@@ -25,3 +25,8 @@ thiserror = "1.0.21"
 
 
 serde = { version = "1.0.136", features = ["derive"] }
+
+[profile.dev]
+# This is required to be able to run `cargo test` in acvm_js due to the `locals exceeds maximum` error.
+# See https://ritik-mishra.medium.com/resolving-the-wasm-pack-error-locals-exceed-maximum-ec3a9d96685b
+opt-level = 1
```

Comment:
I don't think we're currently running the rust tests for `acvm_js` in CI.

Reply:
Is it not beneficial to have this around?

Reply:
It's beneficial to have around as otherwise `cargo test` in `acvm_js` will fail with a `failed to deserialize wasm module` error.

The issue is `cargo test` from the monorepo root doesn't run the rust tests in acvm_js so these could silently fail. We should add another CI job to run that.

Reply:
https://github.com/noir-lang/acvm/issues/475